src/main/java/base/Base.java

```diff
@@ -0,0 +1,24 @@
+package base;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.concurrent.TimeUnit;
+
+public class Base {
+    public static WebDriver driver;
+
+    public static void initialization() {
+        System.setProperty(
+                "webdriver.chrome.driver",
```

Comment:
why do you write these parameters on separate line each? it's not needed and it makes your code hard to understand